authors/apps/articles/views.py

```diff
@@ -390,3 +399,79 @@ def retrieve(self, request, slug, *args, **kwargs):
         read = len(words) / 279
         return Response({'read_time': round(read)}, status.HTTP_200_OK)
 
+
+class ShareArticleViaEmailApiView(CreateAPIView):
+    """Share via Email"""
+
+    def get(self, request, slug):
+        try:
+            try:
+                Article.objects.get(slug=slug)
+            except ObjectDoesNotExist:
+                message = {"errors": {
+                    "message": [
+                        "Article not found."
+                    ]}
+                }
+                return Response(message, status=status.HTTP_404_NOT_FOUND)
+
+            email = request.GET['email']
+            try:
+                validate_email(email)
+                template = 'email_share_article.html'
+                url = base_url + '/api/articles/{}'.format(slug)
```

Comment:
The `/` before `api` is extra so we have `//` before api in the response

Reply:
There is no trailing slash in the `base_url`. Edited in the `env.sample `